hq/test/schedule/AwsDynamoAlertServiceTest.scala

```diff
@@ -1,114 +0,0 @@
-package schedule
```

Comment:
@Nirvikalpa108 was removing the dynamo tests an intentional change?

Reply:
From the commit message, I think it was failing. Going to take a quick look because I think I've worked on these tests.

Reply:
They're passing on this branch for me if I check it out and re-add the test. Happy to help diagnose if they're failing for you @Nirvikalpa108 , let me know.

Reply:
It's true that this test is for the work that we are re-doing, so it doesn't need to be in `main` (and will be removed by anyway in a subsequent PR to merge these two streams of work).

I agree it's not worth the time to fix them in-place. We won't need Database-backed tests for the logic of the new jobs in general, but we may need a bit of coverage for the Database wrapper. I expect while writing that job we can refer back to this (now deleted) file as a reference.